assets/wizards/seo/views/separator/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
```

Comment:
I also encountered this issue, and the `Separator` screen crashed every time.

Reply:
`wp-html-entities` wasn't explicitly listed as a dependency. I didn't run into this error because some other plugin I had installed was enqueuing it. After de-activating all plugins I was able to reproduce. https://github.com/Automattic/newspack-plugin/pull/435/commits/6c614e34f79d8d28b4a7760939ba7b6cf756e17a.